.github/actions/templates/validateModulePSRule/action.yml

```
@@ -118,14 +118,28 @@ runs:
             skipPassedRulesReport   = $false
           }
 
-          Write-Verbose "Set PS Rule Output with following parameters:`n $($Input | ConvertTo-Json -Depth 10)" -Verbose
+          Write-Verbose "Set PS Rule Output with following parameters:`n $($ParameterInput | ConvertTo-Json -Depth 10)" -Verbose
 
           # Invoke Set PSRule Output Functionality
           $null = Set-PSRuleGitHubOutput @ParameterInput
 
           Write-Output '::endgroup::'
 
-    - name: Output to GitHub job summaries
+    - name: 'Output to GitHub job summaries'
       if: always()
       shell: pwsh
-      run: Get-Content '${{ inputs.templateFilePath}}-PSRule-output.md' >> $env:GITHUB_STEP_SUMMARY
+      run: |
+        # Grouping task logs
+        Write-Output '::group::Output to GitHub job summaries'
+
+        $mdPSRuleOutputFilePath = '${{ inputs.templateFilePath}}-PSRule-output.md'
```

```suggestion
        $mdPSRuleOutputFilePath = Join-Path $env:GITHUB_WORKSPACE '${{ inputs.templateFilePath}}-PSRule-output.md'
```

That's probably a good idea to be explicit